pymc3/step_methods/hmc/base_hmc.py

```diff
@@ -132,6 +133,9 @@ def _hamiltonian_step(self, start, p0, step_size):
 
     def astep(self, q0):
         """Perform a single HMC iteration."""
+        perf_start = time.perf_counter_ns()
```

Comment:
So, [according to the tests](https://travis-ci.org/github/pymc-devs/pymc3/jobs/703830359#L3138), there is no `perf_counter_ns` attribute 🤷‍♂️  
`AttributeError: module 'time' has no attribute 'perf_counter_ns'`

Reply:
The function is [new in Python 3.7](https://docs.python.org/3.7/library/time.html#time.perf_counter_ns).

Redefine at import?
```
if sys.version_info.major <= 3 and sys.version_info.minor <= 7:
    perf_counter_ns = lambda: int(time.perf_counter() * 1_000_000_000)
else:
    perf_counter_ns = time.perf_counter_ns
```